python/ngen_cal/src/ngen/cal/ngen_hooks/ngen_output.py

```diff
@@ -0,0 +1,83 @@
+from ngen.cal import hookimpl
+
+import pandas as pd
+from pandas import DataFrame, Series
+from pathlib import Path
+
+class TrouteOutput():
```

Comment:
nitpick. I need to just get pre-commit hooks setup here so I don't have to be that guy lol.

```suggestion
class TrouteOutput:
```

Reply:
fixed in c33327b72e1c60a42e03cc1d2f7d22f6e745dafe